dashboards-observability/public/components/synthetics/home_view.tsx

```diff
@@ -0,0 +1,296 @@
+      pplServiceRequestor(historyQuery + filterVal + timeVal, "jsonData", setHistoryLogs)
+    }
+  };
+
+  const showHelpFlyout = () => {
+    setAddVizDisabled(true);
+    setHelpIsFlyoutVisible(true);
+  };
+
+  const closeHelpFlyout = () => {
+    setAddVizDisabled(false);
+    setHelpIsFlyoutVisible(false);
+  };
+
+  let helpFlyout;
+  if (isHelpFlyoutVisible) {
+    helpFlyout = <PPLReferenceFlyout module="panels" closeFlyout={closeHelpFlyout} />;
+  }
+
+  // input:
+  //    query: any valid query
+  //    type: what part of the response to parse (e.g. 'data' or 'jsonData')
+  //    result: a function to call with results (e.g. setHistoryLogs)
+  // pplServiceRequestor will essentially take a ppl query and then use the function specified in
+  // {result} to put the result into related components
+  const pplServiceRequestor = async (
+    query: string,
+    type: string,
+    result: React.Dispatch<React.SetStateAction<Object[]>>
+  ) => {
+    console.log(query)
+    await props.pplService
+      .fetch({ query: (query), format: 'viz' })
+      .then((res) => {
+        var tempVal = [];
+        for (const field of Object.keys(res[type])) {
+          tempVal.push(res[type][field])
+        };
+        result(tempVal)
+        console.log(tempVal)
+      })
+      .catch((error: Error) => {
+        // setIsError(error.stack);
+        result([])
+        console.error(error);
+      })
+      .finally(() => {
+        // setIsLoading(false);
+      });
+  };
+
+  // columns for the History table
+  const columns = [
+    {
+      field: "monitorName",
+      name: "Monitor Name",
+      render: (monitor: {} | null | undefined) => (
+        <EuiLink href={`#${monitor}`} target="_blank">
+          {monitor}
+        </EuiLink>
+      ),
+      mobileOptions: {
+        show: false
+      },
+    },
+    {
+      field: "status",
+      name: "Status",
+    },
+    {
+      field: "URL",
+      name: "URL",
+      render: (url: {} | null | undefined) => (
+        <EuiLink href={`${url}`} target="_blank">
+          {url}
+        </EuiLink>
+      ),
+      mobileOptions: {
+        show: false
+      },
+    },
+    {
+      field: "response.status",
+      name: "Status Code",
+    },
+    {
+      field: "startTime",
+      name: "Latest Time Polled",
+      type: "date",
+      render: (date: Date) => {
+        var datet: Date = new Date(date)
+        // needed to convert date into local time, as the index stores times in UTC and this page
+        // incorrectly reads it as local time (while being UTC) so the time will be off by the timezoneoffset
+        var utcdate: Date = new Date(datet.getTime() - (datet.getTimezoneOffset() * 60 * 1000))
+        return utcdate.toString()
+      },
+    },
+  ];
+
+  // will pull from index logs every 5 seconds to update at that interval
+  useEffect(() => {
+    const interval = setInterval(() => {
+      console.log("timeq: " + start)
```

Comment:
Some suggestions in general for a PR:
1. Remove commented code
2. Remove all console.log, instead use console.error
3. Remove extra white-space between lines if not needed